ci/qa/security-tests

```diff
@@ -2,6 +2,13 @@
 
 cd $(dirname $0)/../../
 
-"wget -q https://github.com/fabpot/local-php-security-checker/releases/download/v1.0.0/local-php-security-checker_1.0.0_linux_amd64 -O local-php-security-checker && chmod +x ./local-php-security-checker && ./local-php-security-checker",
+if curl -sL https://github.com/fabpot/local-php-security-checker/releases/download/v1.0.0/local-php-security-checker_1.0.0_linux_amd64 > local-php-security-checker; then
+    chmod +x ./local-php-security-checker
+    ./local-php-security-checker
+    rm ./local-php-security-checker
+else
+    printf 'Curl failed downloading php-security-checker with error code "%d"\n' "$?" >&2
+    exit 1
+fi
```

Comment:
Nice one!

Regarding the commit message: `Fix security-tests` What is fixed? I see improvements in the way the script is run. Next time consider something along the lines: Make security-tests more robust.